EmerisHQ/demeris
# fix: show red max text on unavailable state

## Description

as in issue

Fixes: #1810 

## Feature flags

NA

## Testing

1. Enter super huge amount for a pair like Atom <--> Juno (you get an output amount)
2. Enter super huge amount for a pair like Atom <--> Cro (Error from dagg api)
3. Testing with more pairs and huge amounts.

max button text should be red

## Reviews

@luciorubeens
